src/openfermion/transforms/opconversions/remove_symmetry_qubits.py

```diff
@@ -131,3 +133,23 @@ def edit_hamiltonian_for_spin(qubit_hamiltonian, spin_orbital, orbital_parity):
     qubit_hamiltonian.compress()
 
     return qubit_hamiltonian
+
+
+def remove_indices(symbolic_operator, indices):
```

Comment:
Just a small nit could you add a more detailed description of what this function is doing and type annotations?  A minimal requirement is a description of input parameters and types.  even for a small function like this.

Reply:
Thank you for reviewing. Linting errors has been fixed.